gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/ClouddriverService.java

```diff
@@ -338,6 +338,10 @@ Response getAdhocData(
   @GET("/ecs/secrets")
   List<Map> getAllEcsSecrets();
 
+  @GET("/ecs/getEcsClusterDescriptions/{account}/{region}")
```

Comment:
Since you're already define this as a GET operation, typically we don't add the operation type to the path. For instance, instead of adding `getEcsClusterDescriptions` to the path you might want to update to be `ecsClusterDescriptions` like below:

Current: `@GET("/ecs/getEcsClusterDescriptions/{account}/{region}")` 

to 

Updated: `@GET("/ecs/ecsClusterDescriptions/{account}/{region}")`